blog/2023-06-10-conda-libmamba-solver-rollout.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Conda and the libmamba solver: Roll-out plan 2023"
+slug: "2023-06-10-conda-libmamba-solver-rollout"
+description: |
+    Learn about the upcoming rollout plan for the conda-libmamba-solver, bringing enhanced performance and user experience to conda.
+authors: [jezdez]
+tags: [conda, mamba, solver, libmamba, dependency solver, performance, user experience, conda-forge, pycosat, plugin package, Anaconda Distribution, miniconda, CEP 8, CEP 9]
+
+---
+Over a year ago, the conda team at Anaconda, together with conda developers at Quansight and QuantStack, [announced the release](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) of a new solver for conda based on the solver that the [mamba project](https://mamba.readthedocs.io/) had developed on top of the venerable libsolv project.
+
+The dependency solver is one of the main pillars that makes up the conda package and environment manager (next to the network & IO stack and the command line interface integration, for example) and has been the source of many performance and user experience issues over the years, given the growth of the conda ecosystem. Users of conda-forge and other larger conda channels in particular struggled with the current solver implementation based on pycosat.
+
+[Fast-forward to December 2022](https://www.anaconda.com/blog/conda-is-fast-now); we’ve encouraged the use of the libmamba solver via the separate plugin package “conda-libmamba-solver" as a way to trial the use of the solver and get more feedback from real-world users. In case you haven't had the chance to provide your feedback, please head over to the [conda-libmamba-solver repository on GitHub](https://github.com/conda/conda-libmamba-solver).
+
+## Roll-out Plan
+
+
+We're happy to announce the roll-out plan for the next phase for the solver, to get the conda-libmamba-solver in the hands of more people.
+
+<!-- truncate -->
+
+The June 2023 editions of the [Anaconda Distribution](https://www.anaconda.com/download/) and [miniconda](https://docs.conda.io/en/latest/miniconda.html) installers will include the [conda-libmamba-solver](https://conda.github.io/conda-libmamba-solver/) package by default for new installations.
+
+This will make switching to the solver much easier for end users, as it will require only setting `CONDA_SOVLER=libmamba` (for one-time runs) or `conda config --set solver=libmamba` (for persistently switching via the `.condarc` file) and not require additional installation steps. More details on that in the [conda-libmamba-solver getting started guide](https://conda.github.io/conda-libmamba-solver/getting-started/).
```

Comment:
I'd mention the CLI flag instead of the env var.

```suggestion
This will make switching to the solver much easier for end users, as it will require only adding `--solver=libmamba` (for one-time runs) or `conda config --set solver=libmamba` (for persistently switching via the `.condarc` file) and not require additional installation steps. More details on that in the [conda-libmamba-solver getting started guide](https://conda.github.io/conda-libmamba-solver/getting-started/).
```